### PR TITLE
Chore/meet the team redesign

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

A redesign of the "Meet the Team" page to reflect the changes on [figma](https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=132%3A7898). 

Work Done:
 - Updated the cards of each individual with the new design
 - updated the team.yml file to organize it based on individual in specific titles
 - Updated the columns so now they won't be distorted as the screen size gets smaller
 - created titles for each role.

### CODE REVIEW

File: content/en/meet-the-team.html [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+<div class="meet-us-container">
+  <section class="section page-wide--outer-container-padding">
+    <div class="row">
+      <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+        <h2 class="meet-the-team-heading">Meet the Team</h2>
+        
+        <p>We are a team of passionate people committed to making it easier to access and use government
+          services. We all bring something different to CDS, whether that’s experiences from inside or outside
+          government.</p>
+          <div class="cta-wrapper">
```
Comment (dj2) [inline content/en/meet-the-team.html:16]: nit: Indented too far, should lineup with the <p> on line 13

File: content/fr/meet-the-team.html [hunk 2]
```diff
@@ -2,20 +2,33 @@
...
+
+<div class="meet-us-container">
+  <section class="section page-wide--outer-container-padding">
+    <div class="row">
+      <div class="col-sm-10 col-sm-offset-1 col-xs-12">
+        <h2 class="meet-the-team-heading">Vous aimeriez vous joindre à nous?</h2>
+        <p>Nous sommes une équipe de personnes passionnées et déterminées à rendre les services gouvernementaux
+          accessibles et faciles à utiliser. Nous apportons tous quelque chose de différent au SNC, qu’il
+          s’agisse d’expériences internes ou externes au gouvernement.</p>
+          <div class="cta-wrapper">
```
Comment (dj2) [inline content/fr/meet-the-team.html:18]: ditto

File: layouts/section/meet-the-team.html [hunk 3]
```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
...
+            <img class="lazy" src="/img/cds/placeholder.gif" data-src="{{ $baseURL}}{{ $member.imagehash }}.jpg" data-srcset="{{ $baseURL}}{{ $member.imagehash }}.jpg" alt="{{ index $member "name" }}">
+          </div>
+
+          <div class="text">
+            <span class="name">{{ $member.name }}</span>
+            <span class="title">{{ index $member.title (string ( $.Site.Language) ) }}</span>
+            <div class="social-links contact-info-social">
+              {{ if and (ne $member.email nil) (ne $member.email "") }}
+              <span class="social-icons">
+                <a href="mailto:{{ $member.email }}" aria-label="Link to {{ $member.name }}'s Email">
```
Comment (dj2) [inline layouts/section/meet-the-team.html:87]: Does the aria-label need to be translated?

File: layouts/section/meet-the-team.html [hunk 4]
```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
...
+                  <i class="icon-envelope" aria-hidden="true"></i>
+                </a>
+              </span>
+              {{ end }}
+              {{ if and (ne $member.twitter nil) (ne $member.twitter "") }}
+              <span class="social-icons">
+                <a href="https://twitter.com/{{ $member.twitter }}"
+                  aria-label="Link to {{ $member.name }}'s Twitter account">
```
Comment (dj2) [inline layouts/section/meet-the-team.html:95]: Ditto translation

File: layouts/section/meet-the-team.html [hunk 5]
```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
...
+                <a href="https://linkedin.com/in/{{ $member.linkedin }}"
+                  aria-label="Link to {{ $member.name }}'s LinkedIn Profile">
+                  <i class="icon-linkedin" aria-hidden="true"></i>
+                </a>
+              </span>
+              {{ end }}
+              {{ if and (ne $member.github nil) (ne $member.github "") }}
+              <span class="social-icons">
+                <a href="https://github.com/{{ $member.github }}"
+                  aria-label="Link to {{ $member.name }}'s GitHub profile">
```
Comment (dj2) [inline layouts/section/meet-the-team.html:111]: ditto

File: layouts/section/meet-the-team.html [hunk 6]
```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
...
     </div>
-  </div>
 
-  <h2 class="section--title">{{i18n "meet-the-full-team"}}</h2>
+    <div class="heading"> 
+      <h2>{{ i18n "design" }}</h2>
+    </div>
+    <div class="row team-listing wb-eqht">
+      {{ range $member := .Site.Data.team.team.design  }}
+      {{ $v1 := not (isset $member "archived") }}
```
Comment (dj2) [inline layouts/section/meet-the-team.html:72]: Can we indent this block to show it's part of the range loop?

**Review by dj2 — COMMENTED** [review]
When the page gets down to phone size, it feels like the text at the top should extend full width, but is instead constrained to what looks like 2/3 of the width?

File: layouts/section/meet-the-team.html [hunk 7]
```diff
@@ -60,16 +63,470 @@ <h2 class="section--title">{{ i18n "key-contacts" }}</h2>
...
+      {{ end }}
+    </div>
 
-  <div class="container">
+    <div class="heading"> 
+      <h2>{{ i18n "design_research" }}</h2>
+    </div>
+    <div class="row team-listing wb-eqht">
+      {{ range $member := .Site.Data.team.team.design_research  }}
+      {{ $v1 := not (isset $member "archived") }}
```
[the remark this answers is not in the document]
Comment (omartehsin1) [reply layouts/section/meet-the-team.html:129]: Sorry yes, still a WIP, needed to check to see if the deployment link was working.

[the remark this answers is not in the document]
Comment (dj2) [reply layouts/section/meet-the-team.html:129]: Oops, sorry.

File: layouts/section/meet-the-team.html (hunk 3 above)
Comment (omartehsin1) [reply layouts/section/meet-the-team.html:87]: We don't have translations for this, this is from `main`

Comment (dj2) [reply layouts/section/meet-the-team.html:89]: It _should_ be translated though, should it? What does the aria-label say on the French site, is it in English?

Comment (omartehsin1) [reply layouts/section/meet-the-team.html]: Yes, it's in English. Still says "Link to..."

**Comment by omartehsin1** [comment]
Bug for aria label translation issue: https://github.com/cds-snc/digital-canada-ca/issues/2391

File: layouts/section/meet-the-team.html [hunk 8]
```diff
@@ -2,122 +2,63 @@
...
-          </div>
-        </div>
-      </div>
-      {{end}}
-      {{ end }}
+    {{ partial "meet-the-team-partials.html" (dict "team" $.Site.Data.team.exec "lang" (string ( $.Site.Language) ))}}
+
+
+
+    <div class="heading">
```
Comment (dj2) [inline layouts/section/meet-the-team.html:16]: Do these need the `div`s? Seems weird to wrap a `h` tag in a `heading` class div.

**Review by dj2 — COMMENTED** [review]
One question, but otherwise LGTM. Will leave approval for the website folks.

File: layouts/section/meet-the-team.html (hunk 8 above)
Comment (omartehsin1) [reply layouts/section/meet-the-team.html]: Good point, fixed.

**Review by katedee — COMMENTED** [review]
Looking great so far, just a few small things:

- The purple header should be in alignment with the rest of the content
image

- The margin between sections should be a bit larger (so between the last cards of senior management and the Design header, for example)
image

- The cards are a bit narrower in the designs. Should be about 6 in a row, but if that looks too tight, I think 5 would still work on desktop.
image

File: assets/sass/cds/_components.scss [hunk 9]
```diff
@@ -165,12 +170,17 @@
...
+
 	.team-member {
 		position: relative;
-		border-bottom: $primary-color 4px solid;
+		border-bottom: $team-purple 4px solid;
 		box-shadow: rgb(225, 228, 231) 0px -1px 4px;
+		background-color: $medium-grey;
 		padding: 2.5rem 0 0;
 		margin-bottom: 2rem;
+		height: 482px;
```
Comment (katedee) [inline assets/sass/cds/_components.scss:182]: we might want to bump this height down a bit for mobile only, but I leave that up to others

**Review by victoriachanimal — COMMENTED** [review]
Thanks so much @omartehsin1 ! Looks awesome on different screen sizes, and love the icon hover styles. Will defer to @katedee for any additional design tweaks.

**Comment by amazingphilippe** [comment]
Just noticed the bottom margin pushes some social icons out of the card. Maybe a wrapping flex layout could be more robust and help solve this issue?

to align the bottoms of the cards, you can use another wrapping flex layout that automatically stretches the cards to be equal in height in a row. 

Capture d’écran, le 2021-07-09 à 10 26 20

**Comment by amazingphilippe** [comment]
Just one tiny last note if I may: The cards don't need to have artificial blank space below the icons. Some cards will because they don't have social media icons, some won't have any because they are the tallest card in their row. 
Capture d’écran, le 2021-07-09 à 10 31 24

**Comment by dj2** [comment]
> Just one tiny last note if I may: The cards don't need to have artificial blank space below the icons. Some cards will because they don't have social media icons, some won't have any because they are the tallest card in their row.
> Capture d’écran, le 2021-07-09 à 10 31 24

I'm not sure I understand what needs to change here? Is the issue that there is space under the email option in the 2nd card? Or is it something else?

**Comment by amazingphilippe** [comment]
> I'm not sure I understand what needs to change here?

Removing the 15rem bottom margin on each icon is what I am suggesting.

**Comment by katedee** [comment]
Looking even better, thank you!! Just a couple last things:

The margin between these two elements is too wide now, so let's fix that
image

Loving the lesser height on mobile, can we add just a little more space between the icons and border bottom?
image

**Review by victoriachanimal — APPROVED** [review]

**Pull request merged**